CRM/Civicase/Hook/PreProcess/CaseCategoryWordReplacementsForChangeCase.php

```diff
@@ -50,10 +50,10 @@ private function addWordReplacements(CRM_Core_Form $form) {
    *   Form Object.
    */
   private function setPageTitle(CRM_Core_Form $form) {
-    $pageTitle = $form->get_template_vars('activityTypeName');
+    $pageTitle = ts($form->get_template_vars('activityTypeName'));
```

Comment:
@tunbola can we use an alias for ts here? The reason is that ts($someVariable) might be picked up by the translation service and passing variables directly to it it's not supported by the translation service when building the list of strings. It's ok to use it like this after the strings have been populated, that's why using an alias is ok.

Reply:
As discussed we will do more investigation on this as regards how it works on BE and if there needs to be a fix, we will fix in a central place.